operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -108,7 +108,9 @@ func validateFeatures(values *valuesv1alpha1.Values, _ *v1alpha1.IstioOperatorSp
 	if m == nil {
 		return nil
 	}
-	if m.GetAuto().Value && !g.GetControlPlaneSecurityEnabled().Value {
+	cpse := g.GetControlPlaneSecurityEnabled()
+	cpsEnabled := cpse != nil && cpse.Value
```

Comment:
nit: you can do .GetValue() which is the same

Reply:
Fixed